oshmem/mca/spml/ucx/spml_ucx.h

```diff
@@ -286,9 +286,9 @@ mca_spml_ucx_peer_mkey_get(ucp_peer_t *ucp_peer, int index, spml_ucx_cached_mkey
 {
     *out_rmkey = NULL;
     if (OPAL_UNLIKELY((index >= (int)ucp_peer->mkeys_cnt) ||
-                      (MCA_MEMHEAP_MAX_SEGMENTS <= index) || (0 > index))) {
+                      (mca_memheap_base_max_segments <= index) || (0 > index))) {
```

Comment:
can we compare with memheap_context.n_segments?

Reply:
maybe no need it, since already compare with mkeys_cnt

Reply:
yep, removed